frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx

```diff
@@ -106,7 +81,7 @@ export const ProjectOwners: FC<IProjectOwnersProps> = ({ owners = [] }) => {
                     <StyledOwnerName>
                         <StyledHeader data-loading>Owner</StyledHeader>
                         <StyledUserName data-loading>
-                            {users[0]?.name}
+                            {getOwnerName(owners[0])}
```

Comment:
what if owners is an empty array?

Reply:
Same thing happens as before; we show "system"

Reply:
Or rather: before, we wouldn't show anything (but theoretically it should never be empty). Now we show "system"